tools/dind/Dockerfile

```diff
@@ -20,11 +20,8 @@ ARG PASSWORDLESS_DROPBEAR=false
 COPY config/certs/ /usr/local/share/ca-certificates/
 RUN rm -f /usr/local/share/ca-certificates/.keepme ; update-ca-certificates
 
-RUN apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D \
-	&& echo deb https://apt.dockerproject.org/repo debian-${RELEASE_NAME} main > /etc/apt/sources.list.d/docker.list \
-	&& apt-get update || true \
-	&& apt-get --force-yes install docker-engine=${DOCKER_VERSION}-0~${RELEASE_NAME} \
-	&& rm -rf /var/lib/apt/lists/*
+RUN curl -sL "https://github.com/resin-os/balena/releases/download/${BALENA_VERSION}/balena-${BALENA_VERSION}-x86_64.tar.gz" \
+	| tar xzv -C /usr/bin --strip-components=1
 
 ENV DIND_COMMIT 3b5fac462d21ca164b3778647420016315289034
 RUN curl -sL https://raw.githubusercontent.com/docker/docker/${DIND_COMMIT}/hack/dind > /usr/bin/dind \
```

Comment:
Shouldn’t you also copy dind from balena?

Reply:
Good point, fixed